routes/adminRoutes.js

```diff
@@ -13,4 +13,7 @@ router.use((req, res, next) => {
 router.route("/get-user-list")
     .get(getUserList);
 
+router.route("/delete-user/:user_id")
+    .patch(deactivateUser);
```

Comment:
For the project we are using PUT instead of patch, even though it works, to follow a standard
```suggestion
    .put(deactivateUser);
```